packages/mobile/src/store/rate-cta/sagas.ts

```diff
@@ -0,0 +1,19 @@
+import { waitForWrite } from 'audius-client/src/utils/sagaHelpers'
+import { call, put, takeEvery } from 'typed-redux-saga'
+
+import { setVisibility } from '../drawers/slice'
+
+import { requestReview } from './slice'
+
+function* displayRequestReviewDrawer() {
+  yield* call(waitForWrite)
```

Comment:
cool! so is the main reason we wrote a saga for this so we can ensure write is avail before opening?

Reply:
Yea, this is just to make sure that the drawer doesn't open while the app is still loading up and showing a bunch of skeletons. Better to ask when there is content lol